### DISCUSSION TITLE
OBCI Discussion/Conversations

### DISCUSSION THREAD

**Discussion by Reeya123** [clinical-biomarkers/OBCI#12] [Ideas]
**Mar 20, 2023**

### **Darren:**
The latest OBCI has a lot of terms added, but I'd very much like to have examples for various classes that have nothing, so that I can make sure the modeling works for everything we might encounter. I'm interested in examples of the following (2-3 examples each should do it). Note that some of these can be combined (for example, a monitoring biomarker that results from a glycan measurement).

   1) physiological characteristic biomarker - things like blood
pressure perhaps?
   2) glycan biomarker
   3) transcript biomarker - something based on measuring transcripts.
Is this done?
   4) activity biomarker - I saw in LOINC that there are some tests that measure activities. These obviously measure things like outputs, but there can be cases where the activity is what is talked about.
   5) monitoring biomarker for disease
   6) monitoring biomarker for exposure
   7) response biomarker
   8) predictive biomarker - actual BEST predictive, not what many papers call predictive (a mistake often made in the OncoMX set)
   9) safety biomarker
10) biomarker of exposure
11) any biomarker of exposure to "immaterial" things, like radiation
12) biomarker of phenotypic abnormality
13) more genetic variation biomarkers
14) radiographic biomarkers (aka image)
15) histological biomarkers - I can't get a handle on what these are.
Some describe them as things like slides or tissue slices, others say it's cells (but what about flow cytometry?)

Some of these can come from the OncoMX set, but they need to be vetted. Almost every one I checked had some issue, to the point that I don't feel comfortable doing anything automated with it. Actually, that's good advice for any source, as I've seen errors in MarkerDB as well. I'd also like pointers to biomarker resources that can be processed automatically, particularly if they have the following information:

1) the change (increased, etc)
2) the entity
3) the biospecimen - especially in cases of genetic variation (what precisely gets tested?)
4) the disease/phenotype
5) the application - that is, used for diagnosis, monitoring, etc (the BEST category). I've seen statements, for example, to the effect that
all genetic variation biomarkers are diagnostic, prognostic, and risk. Can that be verified?

I'm afraid I don't have the bandwidth to address all of these myself, so any help will be appreciated. I think once we have a decent
representative sample, we can start moving toward the next steps.

**Comment by Reeya123** [comment]
### **Raja:** 
Yes, we can do this

**Comment by DaniallMasood** [comment]
Update 6/16/2023:

Darren received examples which Dan provided. It was an extensive list but not detailed in the OBCI format. I am working on providing examples that will be in that format so Darren can go through them.

**Comment by DaniallMasood** [comment]
Update 7/27/2023

Daniall:
I included some notes in the notes section with some of my thoughts and I would like to know what you think as well. I tried to get an enzymatic biomarker, predictive biomarkers, physiological biomarker, and a biomarker of exposure as well. I reasoned through why they could be those but would want to see what you think. (file attached)


[biomarker examples.xlsx](https://github.com/biomarker-ontology/biomarkerontology/files/12186867/biomarker.examples.xlsx)


Darren:
Thank you for the examples. I'm in the middle of something right now so I'll be able to give more thoughts tomorrow. For now, I'll try to give a distinction between an enzyme as assessed entity vs enzyme activity as assessed entity, and where to find definitions for the BEST biomarker types.

The definitions are easy -  they are in the ontology file. As the definitions given there are the same as the BEST resource, you can also look at https://www.ncbi.nlm.nih.gov/books/NBK338448/def-item/glossary.biomarker/ and (for details) https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5813875/. Note especially how 'predictive' biomarkers are defined; these are ONLY for exposures to chemicals or drugs.

The enzyme vs activity is a nuanced difference. Measure of an enzyme can be done by either measuring the entity itself (protein or complex) or by measuring its activity. The latter aspect itself has two possibilities: (1) measuring the level of reaction product(s) or input(s)s; or (2) measuring the activity using some "black box" method. I'm interested in that last one, as these are the most difficult to find. These will be cases where--for example--some kit does the work and simply reports in terms of activity (perhaps as a percentage against normal?). In these cases we don't actually know what the material entity is.

 Daniall:
Will continue to work on these examples using this new information. Set up meeting with Darren soon to talk this out and hear more of the explanation

**Comment by DaniallMasood** [comment]
Update July 31, 2023:

Darren: 
aspartate transaminase - I looked into what gets measured, and it's always stated that it's 'level of AST'. To me the wording implies a measurement of the actual entity, like what might be determined using quantitative immunoblotting or some method that produces quantity of material. However, when I looked at how the measure is reported, it's given as international units, which are measures of activity. But these units are also defined with respect to quantity, which is to say that activity and quantity can be interconverted. Thinking further on it, and with awareness that measurement techniques can change, I think we need to treat material quantity and activity as somewhat equivalent. That is, in contrast to what I originally thought might be a new assessed entity type, 'activity', I think we could roll that into the relation indicated_by_difference_in_level_of (OBCI:1000014) and its children. Right now that relation explains that 'level' can refer to raw counts, percentages, ratios, and concentrations. I can add 'activities' to this.

That being said, in this case we know the nature of the entity that is being measured (a protein) even if we are measuring it using activity as a proxy. We should be on the lookout for cases where we *don't* know this. I guess this is the sort of example I'm looking for for 'activities'; those where the assessed entity isn't something necessarily material. Right now something like "brain wave activity" came to mind. For example, it could be 'decreased brain wave activity' indicated_by_below_normal_level_of 'brain wave activity', but I'm not sure what the assessed entity type would be. More examples like this would be useful.

HER2/neu protein - The biomarker_status (which I presume is simply what we call 'biomarker') should be 'increased expression of HER2/neu protein'. It is indeed a predictive biomarker, though not for breast cancer. Rather, elevated expression in breast cancer patients will help predict the effectiveness of therapies for those patients.

atrial fibrillation - This would be a risk biomarker based on the note you gave, not predictive (which assesses likelihood of a response to therapy). That is, presence of afib means there is an increased risk of having a stroke (actually having the stroke is not part of the definition of risk biomarkers). I note that you left the specimen type blank. This is good, because that means it MIGHT be an example of what I'm thinking would be a physiological biomarker: biomarkers for which there's no real specimen, or that arise from system-level considerations, or that are themselves medical conditions of interest. I often consider 'increased blood pressure' to be a good candidate, but since it has all three characteristics given above it doesn't help narrow things down. More examples like this would be useful.

3-phenoxybenzioc acid - from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5129381/, "The urinary metabolite, 3-phenoxybenzoic acid (3-PBA), is commonly used to assess human exposure to a number of pyrethroids." This tells you that it is a pharmacological/response biomarker, and that the exposure agent of interest is one of the pyrethroids (ingredient in some insecticides). That's the least we can say with confidence. I'm less certain about the Parkinson's connection, but if there was one that would make 3-PBA a risk biomarker, as you stated. However, the assessed biomarker entity type would be molecular--specifically, a small molecule biomarker.

**Comment by DaniallMasood** [comment]
My question to Darren:

- aspartate transaminase - I understand what you are saying about activity biomarkers and having a parent branch to link these together. I was also looking at this is an enzymatic biomarker. Were you also referring for enzyme biomarkers to come under activity biomarkers as well? How would measuring protein levels as products of an enzyme be classified?

**Reply by DaniallMasood** [reply]
Darrens email reply (August 2): 
To follow my thinking here, you'll need to know one of the ways I envision the ontology could be used. Imagine some doctor orders a test, and the test says that the patient's glucose level is 100. The doctor knows that this is way above normal (I don't really know the numbers; I'm just making these up for illustration). Then the doctor could look at the ontology to find 'elevated glucose level' and from there determine that such findings can mean the chance of X, Y, or Z disease. In this case, the assessed biomarker entity is glucose. When it comes to enzymes, there are multiple ways their levels can be measured. For example, the amount of the enzyme itself can be measured, or the amount of some reaction output can be measured. Indeed, the measurement could even be on something further downstream. In all these cases, we can say what entity is being measured. However, when I looked at some specific examples, I came across one case (which I no longer remember) where the measured entity was reported merely as 'activity', as in "we saw an increase in unicornase activity." So, in my mind I thought "ah, okay, the assessed biomarker entity in this case is an activity" and that led me to think we need--along with things like 'cellular biomarker' (in which the assessed biomarker entity happens to be a cell) and 'molecular biomarker' (in which the assessed biomarker entity happens to be a molecule)--that we'd likewise need to handle cases where the assessed biomarker entity happens to be an activity. However, as I was thinking about my answer in the previous email, I realized that the assessed biomarker entity when using activities is still really just the enzyme level. It's just a proxy. For that reason I thought we don't really need activity as an assessed biomarker type--at least not for enzymes. So, enzymes would not be good candidates for anything called 'activity biomarker'. But then I thought that perhaps something like brain wave activity might be eligible. I'm still not sure if activity biomarker is useful, nor am I even sure that 'brain wave activity' would be a candidate, hence why I wanted more examples to think about.

**Reply by DaniallMasood** [reply]
**Zoom meeting August 3rd**

Darren provides further explanation on enzymes and activity biomarkers:
Enzyme is a molecular biomarker in nature. Usually what is measured is the product of the enzyme or the enzyme itself to deduce the activity of the enzyme. So the enzyme should not be consider an enzyme. Activity biomarkers should have raw levels being measured.

**Comment by DaniallMasood** [comment]
Question to Darren:
HER2/neu - as a predictive biomarker would this be a biomarker for any cancer? we could in the end specify that expression of this in breast cancer links it to therapies in breast cancer. We do have the same biomarker linked to multiple diseases so this would not be difficult to do in our model.

**Reply by DaniallMasood** [reply]
**Darrens Reply August 2**
A predictive biomarker is NEVER for a disease. That is, such biomarkers are not used to predict whether or not someone is likely to get some disease of interest (see NOTE below). Rather, they are used to predict the effectiveness of disease therapies. Scientists, unfortunately, tend to use the word 'predictive' in its more natural language sense when they actually mean something else clinically. Thus, your question would need to be "can elevated expression of HER2/neu predict the effectiveness of <some therapy> on <some cancer>?" I don't know the answer to that, though based on your statement about this being linked to multiple diseases, my guess is that there is at least a possibility that the answer to the revised question is 'yes'. I wouldn't assert it though.

**Reply by DaniallMasood** [reply]
**Darrens Note**
NOTE: Indeed, all cases within https://data.oncomx.org/allbiomarkers that have 'predictive' in the BEST biomarker type column and a disease in the Disease name column are either incorrect or are misleading. The incorrect ones are those where the disease is indeed the concern; as mentioned before diseases are never 'predicted' by a biomarker. The misleading ones are cases where the predicted is actually correct, but what makes them misleading is that the prediction is not for the disease per se but for the treatment of the disease. An example of the former is the [A0105](https://data.oncomx.org/allbiomarkers/biomarker/A0105) with 'predictive' in the BEST column ('predictive' should be 'diagnostic'), while an example of the latter is [A0254](https://data.oncomx.org/allbiomarkers/biomarker/A0254), where the biomarker indicates potential effectiveness of treatment with anthracycline and taxane-containing neoadjuvant chemotherapy in the context of breast cancer. The problem here is that the table is not set up to properly that information. Thus even all correct indications of predictive (or safety or response) biomarkers will be incomplete with respect to what is being asserted in the paper.

**Reply by DaniallMasood** [reply]
**Zoom meeting Aug 3rd**

More clarification on biomarker type. OncoMX data tables have predictive, safety and pharamcodynamic/response biomarkers are mislabeled as these biomarkers are not fully correlated to the disease. They are correlated with the treatments or exposure agents of the disease. Diagnostic, monitoring, prognostic, and risk biomarkers are correlated with disease.  OncoMX needs a way to enter treatment and exposure agent for biomarkers. This would help with the overall biomarker definition.

**Comment by DaniallMasood** [comment]
atrial fibrillation - would any physiological biomarkers such as this be predictive or anything else other than a risk biomarker? Since the presence of this biomarker would just be a risk of having a heart disease.

**Reply by DaniallMasood** [reply]
Darrens Reply August 2

Without other examples I can't say. So far, what I lean toward calling 'physiological biomarkers' do indeed include those that themselves are medical conditions, as in "having this medical condition means there's a risk of having that other medical condition." But I don't think this is necessarily the case. In the ontology itself, right now, we do have 'voice' as an assessed entity, and I have that as a physiological characteristic biomarker type.

**Reply by DaniallMasood** [reply]
**Zoom meeting August 3rd**

Physiological biomarkers have some things in common. Such as they are noninvasive in nature and do not need something to be extracted to measure them. Brainwave could be activity or physiological biomarker, need some more research into this.

**Comment by DaniallMasood** [comment]
Rajas Comments August 3rd:

Daniall - Darren has recommended edits to our table on OncoMX. It might be useful to take that table and put in google doc where each row is a section. You would need to write a script to do this. That way he, you and I can edit ( as tracked/suggested changes). This will allow us to have at least one clean set where all three of us agree. Also, have a curated/organized google drive for the project (use your gmail account to do this; do not use gwu account) to share files etc. 

*I will begin work on this and link the relevant document here

**Comment by DaniallMasood** [comment]
**Zoom meeting Aug 3rd**

Look through protein information resource using protege to find some more biomarker examples. Phenotypic abnormality may have some examples of biomarkers that are not documented in the OBCI format. Darren was able to show me how to use protege and get to the table using his methods.
